mobile/lib/constants/config.dart

```diff
@@ -80,6 +80,8 @@ class Config {
 
   static String get environmentFile => kReleaseMode ? '.env.prod' : '.env.dev';
 
+  static int get locationChangeRadius => 100;
```

Comment:
is this 100 meters or KMs? @NoahNsimbe 

In case it is meters, I suggest that we move it a bit further so that we minimise messages of "no nearest" air quality data.

Reply:
This is how many metres user moves before we refresh their location.